env/staging/replica_count.yaml

```diff
@@ -63,7 +63,7 @@ metadata:
   namespace: notification-canada-ca
 spec:
   minReplicas: 3
-  maxReplicas: 5
+  maxReplicas: 10
```

Comment:
Oh, that slipped in, but it should be increased back again to `10` to match the value in production.